docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+<h2>How to Contribute:</h2>
```

Comment:
I don't think we need embedded HTML here. Let's use native markdown? https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet